app/controllers/users_controller.rb

```diff
@@ -1,5 +1,10 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
+  end
 
-  def show; end
+  def show
```

Comment:
The user encounters this error when he tries to go to the user's show page.
![show](https://user-images.githubusercontent.com/35504149/174405581-80f5deb5-3b3e-42ac-8c5d-8270ad6b718b.png)